src/layouts/MainLayout/BottomNav.js

```diff
@@ -1,3 +1,28 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import BottomNavigation from '@material-ui/core/BottomNavigation';
+import BottomNavigationAction from '@material-ui/core/BottomNavigationAction';
+import { HomeOutlined, PlayCircleOutline, AccountCircleOutlined, AddCircleOutlineOutlined } from '@material-ui/icons';
+import { useHistory, useLocation } from 'react-router';
+import useScrollTrigger from '@material-ui/core/useScrollTrigger';
+import AppBar from '@material-ui/core/AppBar';
+import Fade from '@material-ui/core/Fade';
+import './BottomNav.css';
+
+function HideOnScroll(props) {
+  const { children, window } = props;
+  // Note that you normally won't need to set the window ref as useScrollTrigger
+  // will default to window.
+  // This is only being set here because the demo is in an iframe.
+  const trigger = useScrollTrigger({ target: window ? window() : undefined });
+
+  return (
+    <Fade appear={true} direction="down" in={!trigger}>
+    {children}
+  </Fade>
+  );
+}
+
 import React from "react";
```

Comment:
@RohankumarM you're importing React twice in same file.

Reply:
Did not notice that. 
Updated. 
Thanks for pointing it out @rajatgupta24.